validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -99,16 +102,35 @@ def broadcast_batch_by_batch_id_request(self, batch_id):
             batch_request,
             validator_pb2.Message.GOSSIP_BATCH_BY_BATCH_ID_REQUEST)
 
-    def broadcast(self, gossip_message, message_type):
+    def broadcast(self, gossip_message, message_type, exclude=None):
```

Comment:
This signature is starting to get complicated enough, that we should probably docstring it. Particularly, we should document this new parameter exclude, as it takes a bit of searching through the code to understand what it is (a list) and what is in it (pairs of a particular order).  Something about that starts to feel a little brittle...